Twig/TwigExtension.php

```diff
@@ -2,7 +2,7 @@
 
 namespace BCC\CronManagerBundle\Twig;
 
-class TwigExtension extends \Twig_Extension
+class TwigExtension extends \Twig_Extension implements Twig_Extension_GlobalsInterface
```

Comment:
This won't work and will cause class not found exception. You need to add leading `\` or a `use Twig_Extension_GlobalsInterface` statement.